modules/miscutil/lib/dbquery.py

```diff
@@ -88,6 +91,9 @@
     CFG_DATABASE_SLAVE_SU_USER = ''
     CFG_DATABASE_SLAVE_SU_PASS = ''
     CFG_DATABASE_PASSWORD_FILE = ''
+    CFG_DBDUMPS_TEMP_DIR = ''
```

Comment:
also the names should have started with `CFG_DATABASE_` in order for these to go directly to the `dbquery_config.py` file